### PR TITLE
Added UG for FlatData in Asp.net core and mvc and committed.

### PR DESCRIPTION
**Pull request by saravananj-sf4375**
Included a UG documentation for File Manager with Flat Data in Asp.net core and mvc.

### CODE REVIEW

File: ej2-asp-core-mvc/file-manager/flat-data.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+control: FlatData
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+{% if page.publishingplatform == "aspnet-core" %}
+
+# Rendering Flat JSON Data in the ASP.Net Core File Manager Component
+
+The File Manager uses a flat data JSON as an array of objects for rendering, eliminating the need to define [ajaxSettings](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.FileManager.FileManager.html#Syncfusion_EJ2_FileManager_FileManager_AjaxSettings) url. To load a folder data as an array of objects, use the File Manager component's [fileSystemData](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.FileManager.FileManager.html#Syncfusion_EJ2_FileManager_FileManager_FileSystemData) object. This means you no longer need to use a separate service provider, as you can integrate services like Google, Amazon, Azure, and others directly into your code using the FileManager's action events.
```
Comment (keerthanaRajendran) [inline ej2-asp-core-mvc/file-manager/flat-data.md:15]: https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.FileManager.FileManager.html#Syncfusion_EJ2_FileManager_FileManager_FileSystemData the casing of property is wrong. Please modify @Saravanan-sf4375

File: ej2-asp-core-mvc/file-manager/flat-data.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+description: Learn here all about Flat Data in Syncfusion ##Platform_Name## File Manager component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: FlatData
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+{% if page.publishingplatform == "aspnet-core" %}
+
+# Rendering Flat JSON Data in the ASP.Net Core File Manager Component
```
Comment (keerthanaRajendran) [inline ej2-asp-core-mvc/file-manager/flat-data.md:13]: Change platform name @Saravanan-sf4375

Comment (saravananj-sf4375) [reply ej2-asp-core-mvc/file-manager/flat-data.md:13]: changed by using the generic title.

File: ej2-asp-core-mvc/file-manager/flat-data.md (hunk 1 above)
Comment (saravananj-sf4375) [reply ej2-asp-core-mvc/file-manager/flat-data.md:15]: updated the changes.

**Review by keerthanaRajendran — APPROVED** [review]

**Pull request merged**